workflow/cases/coupled_free_forecast.yaml

```diff
@@ -13,6 +13,7 @@ case:
     nems_temp_cold: 'medcold_atm_ocn_ice'
     mom6ic_prepared: .false.
     KEEPDATA: NO
+    ic_source: cfsr
```

Comment:
Which initial condition is this for? ATM, OCN, ICE, WAV?

Reply:
@JessicaMeixner-NOAA This is the one that we are using for the coupled experiment now, for all components.

Reply:
@JianKuang-NOAA in the current feature/coupled-crow branch, each component has IC variables: 
CPL_ATMIC - atm IC determination for coupled model 
CPL_OCNIC - ocean IC 
CPL_ICEIC - ice IC
CPL_WAVIC - wave IC 
(See: https://github.com/NOAA-EMC/global-workflow/blob/feature/coupled-crow/jobs/rocoto/coupled_ic.sh for their use) 

I see the purpose of this variable, but I think it needs to have a different name or target value.  For example if the "ic_source" was "CPL" to signify that you want to run jobs/rocoto/coupled_ic.sh to source your ICs that makes sense.  But ic_source=cfsr is very misleading because we need and want each component to be able to have a different IC source.

Reply:
OK. Then let's name it "coupled_ic", just so that it means this will trigger the couple_ic.sh script to take care of the IC stacking, instead of fv3_ic.sh